src/app/components/search/search.component.ts

```diff
@@ -109,6 +114,7 @@ export class SearchComponent implements AfterViewChecked, AfterViewInit {
   }
 
   toggleRadioSelected(type: 'specific' | 'range') {
+    this.form.controls['date_from'].reset();
```

Comment:
shouldn't we also reset the other date fields?

Reply:
When we switch from a specific date to a date range for search, since the first date picker for both options is referenced by formControl='date_from', we need this value to be reset on each toggle otherwise we end up with some strange behaviour

Reply:
Yeah good spot, I think 'date_to' needs to be reset when toggling

Reply:
I've just taken another look, you're right when switching from a date range to specific date and back and a date_to has been specified, the date_to will need to be reset